### PR TITLE
make level finding fuzzier

### PR DESCRIPTION
**Pull request by crbates**

### CODE REVIEW

**Comment by crbates** [comment]
This greatly reduces the number of decays with unassigned levels. The real solution of course is to get ENSDF evaluators to agree on level energies but it's a large enough task it probably needs to wait for nuclear data week 2015.

File: src/data.cpp [hunk 1]
```diff
@@ -1443,8 +1443,7 @@ int pyne::id_from_level(int nuc, double level, std::string special) {
   //by default return input nuc_id with level stripped
   int ret_id = nuc;
   for (std::map<std::pair<int, double>, level_data>::iterator it=nuc_lower;
-  it!=nuc_upper;
-       ++it) {
+  it!=nuc_upper; ++it) {
```
Comment (scopatz) [inline src/data.cpp]: SG: Indentation.

File: src/data.cpp [hunk 2]
```diff
@@ -1452,7 +1451,7 @@ int pyne::id_from_level(int nuc, double level, std::string special) {
       ret_id = it->second.nuc_id;
     }
   }
-  if (min > 1.0)
+  if (min > 3.0)
```
Comment (scopatz) [inline src/data.cpp]: A comment on these magic numbers here would be nice :).

**Comment by scopatz** [comment]
To be clear, this is only an interface adjustment, right.  All of the data is still in the db, and recoverable by enterprising youths :)

**Comment by crbates** [comment]
>To be clear, this is only an interface adjustment, right. All of the data is still in the db, and recoverable by enterprising youths :)

well it's an adjustment to the level data interface that affects the decay data portion of `nuc_data.h5`. It shouldn't affect any existing level assignments but it drastically reduces the number of alpha/beta/ecbp decay children that default to the ground state. This is only important for building a decay scheme and determining coincidence it doesn't affect anything we are currently doing.

**Comment by scopatz** [comment]
OK.  So instead these decay children will go to some higher energy level? Or they won't appear as decay children at all?

**Comment by crbates** [comment]
Maybe an example will help 
before:

```
In [2]: data.alpha_child_byparent(922350000)
Out[2]:
[902310038,
 902310000,
 902310027,
 902310023,
 902310020,
 902310019,
 902310000,
 902310000,
 902310015,
 902310013,
 902310012,
 902310011,
 902310006,
 902310004,
 902310003,
 902310002,
 902310001,
 902310000]
```


after:
```
In [2]: data.alpha_child_byparent(922350000)
Out[2]:
[902310038,
 902310000,
 902310027,
 902310023,
 902310020,
 902310019,
 902310017,
 902310016,
 902310015,
 902310013,
 902310012,
 902310011,
 902310006,
 902310004,
 902310003,
 902310002,
 902310001,
 902310000]
```

16 and 17 used to be mis-assigned to the ground state. there is still one that is not assigned correctly but it is a decay with an unknown branching ratio to a level that isn't reported in the adopted levels dataset.

**Comment by scopatz** [comment]
Ahh I see.  Thanks for the clarification.

**Comment by crbates** [comment]
ping @scopatz

**Comment by scopatz** [comment]
Note that this requires a `nuc_data_make --clean=1`

**Comment by scopatz** [comment]
Your levels have been fuzzified :)

**Pull request merged**